source/docs/articles/drupal/configuring-settings-php.md

```diff
@@ -13,6 +13,9 @@ Drupal 8 sites on Pantheon run an unmodified version of core, bundled with a cus
 For Drupal 6/7, Pantheon uses a variant of Pressflow Drupal to allow the server to automatically specify configuration settings, such as the database configuration without editing settings.php. Permissions are handled automatically by Pantheon, so you can customize settings.php like any other site code.
 
 <div class="alert alert-danger" role="alert"><h4>Warning</h4>
+You should not delete the <code>default.settings.php</code> file as this will render the symlink to your <code>sites/default/files</code> directory invalid.</div>
+
+<div class="alert alert-danger" role="alert"><h4>Warning</h4>

```

Comment:
Thinking we can find a better place for the note? Also, should we include that this is for D7 sites and revise the statement to say the directory cannot be empty? @nataliejeremy @slashzero thoughts?